.github/workflows/build.yml

```diff
@@ -19,5 +19,7 @@ jobs:
       - uses: actions/setup-java@v1
         with:
           java-version: '11'
-      - run: ./gradlew clean
-      - run: ./gradlew build
+      - run: ./gradlew clean build
+      - run: |
+          echo useJitpack=false >> gradle.properties
```

Comment:
Looks working but can be improved.

Let's use `local.properties` for this. We already use them and it's inconsistent that now we also use `gradle.properties`. `local.properties` is better IMO because they are needed when we want adjust building locally, not globally.

We can, for example, replace our current `useLocalProjectorServer=true` local property with something like `projectorServerSource` with values like `local`, `jitpack-VERSION`, `space-VERSION`. And by default (if not specified) it could be `jitpack--SNAPSHOT` (like now).

Anyway, for now, it will be a problem if we use both `useJitpack=false` and `useLocalProjectorServer=true`: the build will fail because dependencies names are different. Please ensure that this case is supported.